cpp/ql/src/Security/CWE/CWE-311/CleartextStorage.inc.qhelp

```diff
@@ -19,15 +19,20 @@ cleartext.</p>
 <example>
 
 <p>The following example shows two ways of storing user credentials in a file. In the 'BAD' case,
-the credentials are simply stored in cleartext. In the 'GOOD' case, the credentials are encrypted before 
+the credentials are simply stored in cleartext. In the 'GOOD' case, the credentials are encrypted before
 storing them.</p>
 
 <sample src="CleartextStorage.c" />
 
+<p>Note that for the good example to work we need to link against the encryption library (in this case libsodium),
```

Comment:
Nitpick: Using 'the' here makes it seem like it's the only choice. Could we use 'an' instead?
```suggestion
<p>Note that for the good example to work we need to link against an encryption library (in this case libsodium),
```

Reply:
Yep, I think that's slightly better, thanks.